non_interactive_map.R

```diff
@@ -0,0 +1,95 @@
+install.packages("ggmap")
+install.packages("readr")
+install.packages("tidyr")
+install.packages("plotly")
+
+
+library(ggmap)
+library(maptools)
+library(maps)
+
+library (readr)
+library(tidyr)
+library(plyr)
+
+
+results <- read_csv("C:/Users/tonyn/Downloads/survey_results_public.csv")
+View(results)
+#importing country codes Paul uploaded to github
+country_codes<- read_csv("C:/Users/tonyn/Downloads/countries.csv")
+#cleaning up data
+country_codes <- country_codes[c(-1)]
+country_codes <- country_codes[c(3,1,2)]

```

Comment:
Its' not clear what this line is doing, so country_codes name here would need to be more specific. That make sense?